tide-whisperer.go

```diff
@@ -170,6 +172,21 @@ func main() {
 
 	router := pat.New()
 
+	router.Add("GET", "/swagger", http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
```

Comment:
@derrickburns What's the reason for adding these endpoints? I see the commits where you state you're explicitly ignoring them, but not why.